### ISSUE TITLE
does map chart belong to training set or validation set

### ISSUE THREAD

**Issue by duongdqq** [AlexeyAB/darknet#6507]
Can anyone help me understand: MAP chart belongs to training set or validation set

**Comment by johnny-mueller** [comment]
The graphic that is generated during the training (chart.png) contains only the training lot. You can use the -map flag to display the mAP in the graphic. The mAP is calculated after every fourth epoch.

**Comment by duongdqq** [comment]
> The graphic that is generated during the training (chart.png) contains only the training lot. You can use the -map flag to display the mAP in the graphic. The mAP is calculated after every fourth epoch.

@johnny-mueller 
Thank you for your answer. Do you know how to draw validation loss and training loss in the same chart

**Comment by johnny-mueller** [comment]
@duongdqq 
As far as I kept it in mind, @AlexeyAB wrote among other issues that the validation loss as well as the test loss are not interesting. Of greater interest is the Accuracy, which is described by the mAP.

**Comment by duongdqq** [comment]
> @duongdqq
> As far as I kept it in mind, @AlexeyAB wrote among other issues that the validation loss as well as the test loss are not interesting. Of greater interest is the Accuracy, which is described by the mAP.

The purpose of drawing both training loss and validation loss is to evaluate whether the model is over fitting or under fitting. From my view, it is a necessary step.

**Comment by johnny-mueller** [comment]
If you train with a training dataset and a other validation dataset you should see overfitting even at the mAP. So if the loss will get smaller and the mAP will not increase or decrease.

A validation loss is currently not implemented, so if you want to use it, you probably have to implement it yourself.
